platforms/core-runtime/logging/src/integTest/groovy/org/gradle/DeprecationHandlingIntegrationTest.groovy

```diff
@@ -289,75 +292,6 @@ class DeprecationHandlingIntegrationTest extends AbstractIntegrationSpec {
         }
     }
 
-    def "prints correct deprecation trace property value to use to display stack traces for deprecation warnings"() {
```

Comment:
💭 Considering this wasn't working as planned before, I think it might be worth adjusting and keeping a test like one of these to make it explicit how the `--stacktrace` and the internal flag interact.

Reply:
What was not working as planned? That the internal flag overrides `--stacktrace`? If you check the file, then you'll see that there is such a test. I deliberately only adjusted one of the tests to switch from `--stacktrace` to the internal flag. There is another one that passes `-s` and asserts that not deprecation stacktraces are printed.